<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CatRotator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_connect">
      <property name="text">
       <string>Connect</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_park">
      <property name="text">
       <string>Park</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget_rotator">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Rotator 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton_go">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_posAz">
          <property name="maximum">
           <number>450</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLCDNumber" name="lcdNumber_posAz"/>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButton_stop">
          <property name="text">
           <string>STOP</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Rotator 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionRotator"/>
    <addaction name="actionSetup"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_CatRotator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Hamlib"/>
   </widget>
   <addaction name="menuConfig"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRotator">
   <property name="text">
    <string>Rotator</string>
   </property>
  </action>
  <action name="actionAbout_CatRotator">
   <property name="text">
    <string>About CatRotator</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Hamlib">
   <property name="text">
    <string>About Hamlib</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="text">
    <string>Setup</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
